Misc/NEWS.d/next/Library/2019-02-08-04-46-58.bpo-35936.q2ArjU.rst

```diff
@@ -0,0 +1,2 @@
+:mod:`modulefinder` has been refactored to use the non-deprecated :mod:`importlib`. Additionally, several long-standing bugs have been fixed.
+Patch by Brandt Bucher.
```

Comment:
I personally do not like it when a changelog makes references to bugfixes without specifying what they are. I think maybe it would be appropriate to add separate news entries for each of the long-standing bugs that were fixed. I don't think there's any rule that says that a given PR must have only one changelog entry, and presumably it would be nice to have one changelog ticket per BPO issue number.

Reply:
I think that makes sense, especially if the `imp` fix is eventually backed out.